Makefile

```diff
@@ -170,11 +169,17 @@ plugin-deps: $(plugin_includes) $(plugin_c_includes)
 # --------
 
 PYTHON_BIN   := python3.10
-KEVM_PYK_DIR := ./kevm-pyk
+KEVM_PYK_DIR := $(KEVM_LIB)/kevm-pyk
 POETRY       := poetry -C $(KEVM_PYK_DIR)
 POETRY_RUN   := $(POETRY) run --
 
-poetry-env:
+kevm_pyk_files := $(shell find kevm-pyk -type f -name '*.py') \
+                  kevm-pyk/Makefile                           \
+                  kevm-pyk/pyproject.toml                     \
+                  kevm-pyk/poetry.lock                        \
+                  kevm-pyk/.flake8
```

Comment:
For development this is not necessary. For distributing in a Docker image, why not install `kevm-pyk` with `pip`? Then

* The `Makefile` does not become more complicated
* `poetry` does not become a runtime dependency
* This will also be unnecessary:

> It requires that the user call `poetry install ... kevm-pyk` before trying to use the installed KEVM.

Reply:
I have switched to doing a user-level install of kevm-pyk in the docker image using pip. I tried to get a global debian install working, to have the standalone package work, but not very hard, because the only deliverable is a docker image, and I figure we can fix that later if needed.